cmd/juju/storage/list.go

```diff
@@ -29,6 +29,20 @@ const listCommandDoc = `
 List information about storage.
 `
 
+const listCommandExample = `
+List all storage:
+
+    juju storage
+
+List only filesystem storage(deprecate):
+
+    juju storage --filesystem
+
+List only volume storage(deprecate):
+
+    juju storage --volume
+`
+
```

Comment:
What is the "(deprecate)" note about?  If the flag is deprecated that info should be given on the flag itself and we shouldn't bother to illustrate it in our examples.